### ISSUE TITLE
Overlay UI when a URL is shared with Seal

### ISSUE THREAD

**Issue by rodrigoswz** [JunkFood02/Seal#152]
### Checklist

- [ ] This feature I'm requesting is already implemented in yt-dlp.
- [X] This feature is merely a UI/UX update.
- [ ] This feature is suitable for primary users with little knowledge about yt-dlp.
- [ ] This feature is available for most websites, not only the video platform I use.
- [ ] This feature is suitable for a large variety of videos.
- [ ] This feature is not going to conflict with many of the existing options.

### Is your feature request related to a problem? Please describe.

Currently when sharing a video URL with Seal, the full app opens. 

### Describe the solution you'd like

It would be much better and visually more pleasant if it was just an overlay with quick settings before starting the download.

**Comment by BobbyESP** [comment]
I thought on this before. Something like just open the "Configure before downloading"  menu?

**Comment by rodrigoswz** [comment]
> I thought on this before. Something like just open the "Configure before downloading"  menu?

Exactly! Just the bottom menu overlapping the video app (YouTube, for example).

**Comment by BobbyESP** [comment]
Understood! The best way to do this i think is just making the MainActivity unvisible

**Comment by ShowcaseKing** [comment]
It could be done like how newpipe does it.

**Comment by JunkFood02** [comment]
This feature has been requested a few times. The difficulty here is that we are using a very new UI framework to build this application, while "system sharing" uses an old-fashioned API, so it may require a UI refactoring to implement. Also, the focus of the next release is on multitasking support, so any other UI changes will be put on hold.

**Comment by JunkFood02** [comment]
Implement a workaround in v1.3.5

**Comment by Marco-9456** [comment]
> Implement a workaround in v1.3.5

Can you please make this optional in the settings

**Comment by JunkFood02** [comment]
Implement in v1.8.0-beta.2
